### PR TITLE
feat: document use of SecCollectionTimeout

### PR DESCRIPTION
**Pull request by theseion**

### CODE REVIEW

File: content/development/plugin_writing.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ What to do is as follows:
 * **Scoring in phase 1:** Put in the plugin's *After-File* (and be aware that early blocking won't work).
 * **Scoring in phase 2:** Put in the plugin's *Before-File*.
 
+### ModSecurity SecCollectionTimeout
+
+CRS does not set change the default value (`3600`) for the ModSecurity `SecCollectionTimeout` directive. Plugin authors must instruct users to set the directive to an appropriate value if the plugin requires a value that differs from the default. A plugin should never actively set `SecCollectionTimeout`, as other plugins may specify different values for the directive and the choice for the effective value must be made by the user.
```
Comment (dune73) [inline content/development/plugin_writing.md:45]: The "not set change" is probably a typo. Other than that I think this is good to go.

**Review by dune73 — CHANGES_REQUESTED** [review]

**Comment by theseion** [comment]
ping @RedXanadu

**Review by RedXanadu — CHANGES_REQUESTED** [review]

**Comment by RedXanadu** [comment]
So, I did not press the correct button to publish the change request… Thank you @theseion!

**Review by RedXanadu — APPROVED** [review]
Awesome, thank you!

**Review by dune73 — APPROVED** [review]

**Comment by RedXanadu** [comment]
Merging now.

**Pull request merged**